packages/babel-plugin-transform-vue-jsx/test/snapshot.js

```diff
@@ -301,6 +301,21 @@ render(h => h("div", _mergeJSXProps([{}, spread, {
   }
 });`,
   },
+  {
+    name: 'Root attribute',
+    from: `<MyComponent propsProp1="foo" props={{ prop1: 'alpha', prop2: 'beta' }} />`,
```

Comment:
I'm assuming `propsProp1="foo"` is same as `props={ prop1: 'foo' }`.

Reply:
Does the generated code depend on prop order?

`<MyComponent props={{ prop1: 'alpha', prop2: 'beta' }}  propsProp1="foo" />`

Reply:
@znck, yes, it will be merged in the end to something that behaves like:
```js
{
  ...{ prop1: 'alpha', prop2: 'beta' }
  ...{ prop1: 'foo' }
}
```
Just like it normally would work if you declare a JS object with same key twice...